texts/MANIFEST

```diff
@@ -3,3 +3,4 @@
 1   icarus      dolphingarlic   ACCESSED FROM: http://www.angelfire.com/dragon/erzulie/Icarus_Allsorts.htm                                      AUTHOR: Roger McGough           STYLE: Poem               LICENSE: Public Domain    YEAR: 1816
 1   kubla_khan  dolphingarlic   ACCESSED FROM: https://www.poetryfoundation.org/poems/43991/kubla-khan                                          AUTHOR: Samuel Taylor Coleridge STYLE: Poem               LICENSE: Creative Commons YEAR: 1968
 1   reggae      dolphingarlic   ACCESSED FROM: https://docs.google.com/document/d/1Rz7DNd8vFPaU_baNtyZVCIxAGUsTcamFnAxHT92AXBU/edit?usp=sharing AUTHOR: Andi Qu                 STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
+1   evolution   nu11us          ACCESSED FROM: https://simple.wikipedia.org/wiki/Evolution                                                      AUTHOR: N/A                     STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
```

Comment:
Please link to the exact version of the page.

Reply:
By exact version, @wei2912 means the id in the history of that page.